MaslennikovKM/lab_4/lab_4.cpp

```diff
@@ -0,0 +1,87 @@
+#include <iostream>
+
+float number_in_degree(float number, int degree) {
+	float result = 1;
+	if ((degree <= 0) && (number == 0)) {
+		std::cout << "Not defined" << std::endl;
```

Comment:
Your functions should not print anything to console
Instead it should return result value